### ISSUE TITLE
Zogre Flesh Eaters Various Issues

### ISSUE THREAD

**Issue by Tazatac** [Zoinkwiz/quest-helper#1093]
1. The required "5 free inventory spaces" is somewhat misleading, you will either need a lot more or need to read through the wiki to see which items can be safely dropped.
2. The last step of the investigation phase does not tell you that you ALSO need to keep the tankard after talking to the barkeep, it only tells you that you need the portrait and two books, which is incorrect.

**Comment by Zoinkwiz** [comment]
For 2, did Zavistic not let you continue without the tankard? Weirdly when I made the guide I explicitely tested without and it all seemed to work fine, so I wonder if something's changed or something.

**Comment by Tazatac** [comment]
Yes exactly, when I approached zavistic without the tankard, he claimed I
didn't have enough evidence. Getting another tankard and returning to him
added a dialogue line of presenting the tankard as the last piece of
evidence, and continued the quest.

Sincerely,

Zac Tyrrell
